test/tests/ver-1ie/comparison_ver-1ie.py

```diff
@@ -0,0 +1,84 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+from scipy import special
+import os
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+script_folder = os.path.dirname(__file__)
+os.chdir(script_folder)
+
+#===============================================================================
+# Physical constants
+kb = 1.380649e-23  # J/K Boltzmann constant
+
+# ===============================================================================
+# Extract TMAP8 results
+if "/TMAP8/doc/" in script_folder:     # if in documentation folder
+    csv_folder = "../../../../test/tests/ver-1ie/gold/ver-1ie_out.csv"
+else:                                  # if in test folder
+    csv_folder = "./gold/ver-1ie_out.csv"
+tmap8_solution = pd.read_csv(csv_folder)
+tmap8_solution_time = tmap8_solution['time']
+tmap8_solution_B2 = tmap8_solution['pressure_B2']
+tmap8_solution_A2 = tmap8_solution['pressure_A2']
+tmap8_solution_AB = tmap8_solution['pressure_AB']
+
+
+# ===============================================================================
+# Calculate analytical solution
+def get_analytical_solution(numerical_steps):
+
+    T = 1000 # K Temperature
+    V = 1.0 # m^3 Volume
+    S = 0.0025 # m^2 Area
+    p0_A2 = 1e4 # Pa Initial pressure for A2
+    p0_B2 = 1e4 # Pa Initial pressure for B2
+    K_d = 1.858e24 / np.sqrt(T) # at.m^-2/s/pa dissociation rate for AB
+    # time_constant = 0.15
+
+    p_AB_analytical_ratedep = 2 * p0_A2 * p0_B2 * (1 - np.exp(-S * K_d * kb * T * numerical_steps / V)) / (p0_A2 + p0_B2)
+    # p_AB_analytical_tao = 2 * p0_A2 * p0_B2 * (1 - np.exp(- numerical_steps / time_constant)) / (p0_A2 + p0_B2)
+    p_AB_analytical_special = 2 * p0_A2 * p0_B2 * (1 - np.exp(- 2 * S * K_d * kb * T * numerical_steps / V)) / (p0_A2 + p0_B2)
+    return p_AB_analytical_special, p_AB_analytical_ratedep
+
+
+p_AB_analytical, p_AB_analytical_ratedep = get_analytical_solution(tmap8_solution_time)
+
+# ===============================================================================
+# Plot concentration evolution as a function of time
+# Recreates TMAP4 verification plot
+
+fig = plt.figure(figsize=[6.5, 5.5])
+gs = gridspec.GridSpec(1, 1)
+ax = fig.add_subplot(gs[0])
+alpha = 0.6
+ax.plot(tmap8_solution_time, tmap8_solution_A2,
+        label=r"$A_2$ TMAP8", c='tab:brown', alpha=alpha)
+ax.plot(tmap8_solution_time, tmap8_solution_B2,
+        label=r"$B_2$ TMAP8", c='tab:cyan', alpha=alpha, linestyle='--')
+ax.plot(tmap8_solution_time, tmap8_solution_AB,
+        label=r"$AB$ TMAP8", c='tab:gray', linestyle='-')
+ax.plot(tmap8_solution_time, p_AB_analytical,
+        label=r"$AB$ Analytical", c='k', linestyle='--')
+# ax.plot(tmap8_solution_time, p_AB_analytical_ratedep,
+#         label=r"$AB$ in Ratedep Analytical", c='tab:blue', linestyle='--')
```

Comment:
```suggestion
```
So I see that you do not include the solution with the adapted tau, why is that?